src/main/java/com/ericsson/gerrit/plugins/eiffel/handlers/MessageQueueHandler.java

```diff
@@ -17,15 +17,15 @@
 
 package com.ericsson.gerrit.plugins.eiffel.handlers;
 
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+
 import com.google.gerrit.extensions.annotations.PluginName;
 import com.google.gerrit.extensions.events.LifecycleListener;
 import com.google.gerrit.server.config.PluginConfig;
 import com.google.gerrit.server.config.PluginConfigFactory;
 import com.google.gerrit.server.git.WorkQueue;
 import com.google.inject.Inject;
 
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-
 public class MessageQueueHandler implements LifecycleListener {
```

Comment:
Can you put a javadoc on how this is used since I can't find a place where we actually add something to the queue.

Reply:
Maybe @a-breid-ericsson can do that in another PR as he wrote this? I don't understand this either.

Reply:
Fixed myself in https://github.com/eiffel-community/eiffel-gerrit-plugin/pull/39